packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/index.js

```diff
@@ -143,11 +143,16 @@ class DrawerTray extends Component {
     shouldReturnFocus: PropTypes.bool,
     shouldCloseOnDocumentClick: PropTypes.bool,
     shouldCloseOnEscape: PropTypes.bool,
-
     // eslint-disable-next-line react/require-default-props
     makeStyles: PropTypes.func,
     // eslint-disable-next-line react/require-default-props
-    styles: PropTypes.object
+    styles: PropTypes.object,
+    // eslint-disable-next-line react/require-default-props
+    dir: PropTypes.oneOf(Object.values(bidirectional.DIRECTION)),
+    // eslint-disable-next-line react/require-default-props
+    rtl: PropTypes.bool,
+    // eslint-disable-next-line react/require-default-props
+    ltr: PropTypes.bool
```

Comment:
Are rtl and ltr props needed or just dir? Also, please add them in the ui-docs-client/Properties `renderRows` method as exceptions, so they don't appear in the docs.

Reply:
Done